### PR TITLE
[DOCFIX] Add documents for running alluxio on tencent cloud EMR

### PR DESCRIPTION
**Pull request by wheatxiong**
This guide describes how to configure Alluxio to run on Tencent Cloud EMR.

fixes https://github.com/Alluxio/alluxio/issues/12692

### CODE REVIEW

**Comment by wheatxiong** [comment]
> alluxio-bot, check this please

**Comment by wheatxiong** [comment]
alluxio-bot, check this please

**Comment by wheatxiong** [comment]
alluxio-bot, check this please

File: docs/cn/cloud/Tencent-Cloud-EMR.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+## 概述
+
+在腾讯云EMR上提供了开箱可用的Alluxio服务，以帮助腾讯云客户可以快速实现分布式内存级缓存加速，简化数据管理等能力；同时还可以通过腾讯云EMR控制台或API接口，使用配置下发功能力快速配置多层级缓存和元数据管理等；获取一站式监控告警能力等能力。
+
+
+## 准备
+
+腾讯云EMR的Hadoop标准版本>=2.1.0
+腾讯云EMR的Hadoop天穹版本>=1.0
+有关EMR中版本中支持具体的Alluxio的版本支持可参考[这里](https://cloud.tencent.com/document/product/589/20279)
```
Comment (apc999) [inline docs/cn/cloud/Tencent-Cloud-EMR.md:22]:
```suggestion
- 腾讯云EMR的Hadoop标准版本>=2.1.0
- 腾讯云EMR的Hadoop天穹版本>=1.0
- 有关EMR中版本中支持具体的Alluxio的版本支持可参考[这里](https://cloud.tencent.com/document/product/589/20279)
```

File: docs/cn/cloud/Tencent-Cloud-EMR.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  ![](https://application-display-1259353343.cos.ap-hongkong.myqcloud.com/alluxio/alluxio-config_cn.png)
+在配置下发后，有些配置需要重启Alluxio服务才能生效:
+  ![](https://application-display-1259353343.cos.ap-hongkong.myqcloud.com/alluxio/alluxio-restart_cn.png)
+
+
+了解跟多配置下发和重启策略细节，可以查阅相关文档:
+
+[配置管理](https://cloud.tencent.com/document/product/589/14628)
+
+[重启服务](https://cloud.tencent.com/document/product/589/32823)
```
Comment (apc999) [inline docs/cn/cloud/Tencent-Cloud-EMR.md:61]:
```suggestion
- [配置管理](https://cloud.tencent.com/document/product/589/14628)
- [重启服务](https://cloud.tencent.com/document/product/589/32823)
```

File: docs/cn/cloud/Tencent-Cloud-EMR.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  
+  ![](https://application-display-1259353343.cos.ap-hongkong.myqcloud.com/alluxio/auth-confirm_cn.png)
+  
+  {% endcollapsible %}
+  
+{% endaccordion %}
+
+
+更多在腾讯云EMR中使用Alluxio开发使用细节，可[查阅](https://cloud.tencent.com/document/product/589)
+
```
Comment (apc999) [inline docs/cn/cloud/Tencent-Cloud-EMR.md:82]:
```suggestion
```

File: docs/en/cloud/Tencent-Cloud-EMR.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+{:toc}
+
+## Overview
+The out-of-the-box Alluxio service provided on Tencent Cloud EMR can help customers quickly achieve distributed memory-level cache acceleration while simplifying data management. At the same time, the Alluxio service can quickly configure multi-level caches and manage metadata by operating the EMR console or API interface using the configuration delivery function, as well as the ability to obtain one-stop monitoring and alarms.
+
+
+## Prerequisites
+Hadoop standard version of Tencent Cloud EMR >= 2.1.0
+Hadoop Skydome version of Tencent Cloud EMR >= 1.0
+For the specific Alluxio version supported in EMR, please refer to [ Component Version supported Tencent Cloud EMR](https://intl.cloud.tencent.com/document/product/1026/31095).
```
Comment (apc999) [inline docs/en/cloud/Tencent-Cloud-EMR.md:21]:
```suggestion
- Hadoop standard version of Tencent Cloud EMR >= 2.1.0
- Hadoop Skydome version of Tencent Cloud EMR >= 1.0
- For the specific Alluxio version supported in EMR, please refer to [Component Version supported Tencent Cloud EMR](https://intl.cloud.tencent.com/document/product/1026/31095).
```

File: docs/en/cloud/Tencent-Cloud-EMR.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  ![](https://application-display-1259353343.cos.ap-hongkong.myqcloud.com/alluxio/cos-auth.png)
+  
+  ![](https://application-display-1259353343.cos.ap-hongkong.myqcloud.com/alluxio/auth-confirm.png)
+
+  {% endcollapsible %}
+  
+{% endaccordion %}
+
+For more details on using Alluxio development in Tencent Cloud EMR, please refer to [here](https://intl.cloud.tencent.com/document/product/1026/31168).
+
```
Comment (apc999) [inline docs/en/cloud/Tencent-Cloud-EMR.md:71]:
```suggestion
```

**Review by apc999 — CHANGES_REQUESTED** [review]
thanks for the contribution!

**Comment by wheatxiong** [comment]
> thanks for the contribution!

Thank you very much for reviewing it ! 

I had changes as requested, pls review again.

**Review by apc999 — APPROVED** [review]
Thanks for the contribution!

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**